docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -41,6 +41,9 @@ From your local workstation, create the secrets:
 $ tctl auth sign --format=db --host=db.example.com --out=server --ttl=2190h
 ```
 
+In this example, `db.example.com` is the hostname that the MySQL server is
+accessible from to the Teleport Proxy service.
```

Comment:
This applies to this page and the others in this PR:

- I think we should use "Proxy Service" instead of "Proxy service" to be more consistent with the rest of the docs/the Core Concepts page
- "accessible from to" comes across a bit awkwardly for me. Maybe, "is the hostname where the Teleport Proxy Service can reach the MySQL server"? Also, is the Teleport Proxy Service dialing MySQL, or is it the Database Service?